docs/reference/method/MongoDBClient-addSubscriber.txt

```diff
@@ -0,0 +1,130 @@
+================================
+MongoDB\\Client::addSubscriber()
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Client::addSubscriber()
+
+    Registers a monitoring event subscriber with this Client. The subscriber
+    will be notified of all events for this Client.
+
+   .. code-block:: php
+
+      function addSubscriber(MongoDB\Driver\Monitoring\Subscriber $subscriber): void
+
+   This method has the following parameters:
+
+   .. include:: /includes/apiargs/MongoDBClient-method-addSubscriber-param.rst
+
+   .. note::
+
+         If $subscriber is already registered with this Client, this function
+         is a no-op. If $subscriber is also registered globally, it will still
+         only be notified once of each event for this Client.
+
+Errors/Exceptions
+-----------------
+
+.. include:: /includes/extracts/error-invalidargumentexception.rst
+
+Example
+-------
+
+Create a :phpclass:`MongoDB\\Driver\\Monitoring\\CommandSubscriber` that
+logs all events:
```

Comment:
I created this example as the example on [Application Performance Monitoring (APM)](https://www.php.net/manual/en/mongodb.tutorial.apm.php#mongodb.tutorial.apm) seemed complicated to me

Reply:
I update the example to use the client method. Here is the ticket for the tutorial: https://jira.mongodb.org/browse/PHPLIB-1314